lib/tasks/openapi_generate.rake

```diff
@@ -0,0 +1,388 @@
+class OpenapiGenerator
+  def rails_routes
+    Rails.application.routes.routes.each_with_object([]) do |route, array|
+      r = ActionDispatch::Routing::RouteWrapper.new(route)
+      next if r.internal? # Don't display rails routes
+      next if r.engine? # Don't care right now...
+
+      array << r
+    end
+  end
+
+  def openapi_file
+    Pathname.new(__dir__).join("../../public/doc/openapi-3-v0.2.0.yaml").to_s
```

Comment:
This feels weird that we would hardcode the 0.2.0 version in the generator...I'd expect it as a parameter or something.

Reply:
Agreed. version is now generated from the latest openapi.json route defined for the project.